CMakeLists.txt

```diff
@@ -13,8 +13,16 @@ endif ()
 include(cmake/compilation_flags.cmake)
 list(APPEND GLOBAL_CXXFLAGS ${DEFAULT_CXX_FLAGS})
 
+option(BUILD_SHARED_LIBS "Build Shared Libraries" OFF)
+option(BUILD_EXAMPLES "Build Examples" ON)
```

Comment:
We already have an *examples* folder, which is not built automatically; the *dump-pe* executable is more of a utility than an example. Can we rename this variable to something similar to "Build command line tools"?

Reply:
I wasn't sure after reading https://github.com/trailofbits/pe-parse/pull/52#issuecomment-320362425

> dump-prog is more an example than an end user program

Reply:
Changed to command line tools and rebased